vector/src/gplay/java/im/vector/app/GoogleFlavorLegals.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2022 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.app
+
+import android.content.Context
+import android.content.Intent
+import com.google.android.gms.oss.licenses.OssLicensesMenuActivity
+import im.vector.app.features.settings.legals.FlavourLegals
+import javax.inject.Inject
+
+class GoogleFlavorLegals @Inject constructor() : FlavourLegals {
```

Comment:
Could we rename `FlavourLegals` to `FlavorLegals` to be consistent in naming?

Reply:
will do :+1:

Reply:
https://github.com/vector-im/element-android/pull/6783/commits/439224e4dec4ec1314b5f15e315201a996961730